pkg/volume/csi/nodeinfomanager/nodeinfomanager_test.go

```diff
@@ -75,12 +74,17 @@ func TestInstallCSIDriver(t *testing.T) {
 			inputTopology: map[string]string{
 				"com.example.csi/zone": "zoneA",
 			},
+			expectedCSINode: generateCSINode(
```

Comment:
Instead of using generateCSINode, can you just put the direct CSINode object, like [here](https://github.com/kubernetes/kubernetes/pull/75481/files#diff-562044ebf082e2a220401c0a69b9a58eR183)?  This will make it more obvious what the expected output is supposed to be.